docs/sections/contributor_guide/fork_pr_model.rst

```diff
@@ -4,15 +4,15 @@ Fork and PR Model
 Overview
 --------
 
-Contributions to the ``workflow-tools`` project are made via a `Fork <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks>`__ and `Pull Request (PR) <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests>`_ model. GitHub provides a thorough description of this contribution model in their `Contributing to projects` `Quickstart <https://docs.github.com/en/get-started/quickstart/contributing-to-projects>`_, but the steps, with respect to ``workflow-tools`` contributions, can be summarized as:
+Contributions to the ``workflow-tools`` project are made via a :github-docs:`Fork<pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks>` and :github-docs:`Pull Request (PR)<pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests>` model. GitHub provides a thorough description of this contribution model in their `Contributing to a project` :github-docs:`Quickstart<get-started/exploring-projects-on-github/contributing-to-a-project>`, but the steps, with respect to ``workflow-tools`` contributions, can be summarized as:
 
-#. `Fork <https://docs.github.com/en/get-started/quickstart/contributing-to-projects#forking-a-repository>`__ the ``workflow-tools`` `repository <https://github.com/ufs-community/workflow-tools>`_ into your personal GitHub account.
-#. `Clone <https://docs.github.com/en/get-started/quickstart/contributing-to-projects#cloning-a-fork>`_ your fork onto your development system.
-#. `Create a branch <https://docs.github.com/en/get-started/quickstart/contributing-to-projects#creating-a-branch-to-work-on>`_ in your clone for your changes.
-#. `Make, commit, and push changes <https://docs.github.com/en/get-started/quickstart/contributing-to-projects#making-and-pushing-changes>`_ in your clone / to your fork. (Refer to the :doc:`Developer Setup <developer_setup>` page for setting up a development shell, formatting and testing your code, etc.)
-#. When your work is complete, `create a pull request <https://docs.github.com/en/get-started/quickstart/contributing-to-projects#making-a-pull-request>`_ to merge your changes.
+#. :github-docs:`Fork<get-started/exploring-projects-on-github/contributing-to-a-project#forking-a-repository>` the :uwtools:`workflow-tools reposiroty<>` into your personal GitHub account.
```

Comment:
Typo: "reposiroty" instead of "repository"

Reply:
So many typos this time! My editor has a spellchecker; I should use it. Thanks for noticing these. Fixed.